ctru-rs/examples/audio-filters.rs

```diff
@@ -79,10 +79,7 @@ fn main() {
     channel_zero.set_format(AudioFormat::PCM16Stereo);
 
     // Output at 100% on the first pair of left and right channels.
```

Comment:
This comment might be good to move to the `Default` impl itself since the example is no longer specifying it explicitly.